noxfile.py

```diff
@@ -235,7 +248,12 @@ def test_next(
 
     nox_utils.install_session_venv(
         session,
-        extras=["performance", "testing", *codegen_settings["extras"], *device_settings["extras"]],
+        extras=[
+            "performance",
+            "testing",
+            *codegen_settings["extras"],
+            *device_settings["extras"],
```

Comment:
Did you add the trailing comma here on purpose? If not, you can try removing the trailing comma again to undo the automatic formatting
```suggestion
            *device_settings["extras"]
```